tartufo/util.py

```diff
@@ -66,9 +66,18 @@ def echo_result(
             "excluded_signatures": [
                 str(signature) for signature in options.exclude_signatures
             ],
-            "found_issues": [issue.as_dict() for issue in scanner.issues],
+            "found_issues": [
+                issue.as_dict(compact=options.compact) for issue in scanner.issues
+            ],
         }
+
         click.echo(json.dumps(output))
+    elif options.compact:
+        for issue in scanner.issues:
+            click.echo(
+                f"[{issue.issue_type.value}] {issue.chunk.file_path}: {issue.matched_string} "
```

Comment:
not sure if this trailing space is intentional

Reply:
Yup! This is using implicit string joining to the next line, so these lines end up looking like (extracting from the tests):
> [Regular Expression Match] /what/bar: bar (fa692eebc3d60e67a9f22b4b877d5939cb2ec96c0c26c7e5168b3b8b660c573c, Meets the bar)